cmd/executor/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/kubeflow/kfserving/pkg/executor"
+	"github.com/pkg/errors"
+	"golang.org/x/net/http2"
+	"golang.org/x/net/http2/h2c"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var (
+	predictor   = flag.String("predictor", "", "The FQDN of the predictor service")
+	preprocess  = flag.String("preprocess", "", "The FQDN of the preprocess service")
+	postprocess = flag.String("postprocess", "", "The FQDN of the postprocess service")
+	port        = flag.Int("port", 8080, "Executor port")
+)
+
+func main() {
+	flag.Parse()
+
+	logf.SetLogger(logf.ZapLogger(false))
+	log := logf.Log.WithName("entrypoint")
+
+	stopCh := signals.SetupSignalHandler()
+
+	var eh http.Handler = executor.New(log, *preprocess, *predictor, *postprocess)
```

Comment:
should we need check if  preprocess, predictor, postprocess are valid?
for example `predictor` at least cannot be empty.

Reply:
Done